molecule/default/prepare.yml

```
@@ -1,8 +1,21 @@
 ---
 - name: Prepare
   hosts: all
-  tasks:
-
+  vars:
+    mysql_root_username: root
+    mysql_root_password: netways0815!
```

```suggestion
    mysql_root_password: ChangeMe
```
Please use a more common password for tests